hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/classic/ClassicOperationExecutor.java

```diff
@@ -439,18 +436,13 @@ private static void awaitTermination(OperationThread[] operationThreads) {
 
     @Override
     public void dumpPerformanceMetrics(StringBuffer sb) {
-        for (int k = 0; k < partitionOperationThreads.length; k++) {
-            OperationThread operationThread = partitionOperationThreads[k];
-            sb.append(operationThread.getName())
-                    .append(" processedCount=").append(operationThread.processedCount)
-                    .append(" pendingCount=").append(operationThread.scheduleQueue.size())
-                    .append('\n');
+        for (PartitionOperationThread operationThread : partitionOperationThreads) {
+            sb.append(operationThread.getName()).append(" processedCount=").append(operationThread.processedCount)
```

Comment:
Perhaps there is an issue with your IDE. BUt when I select the original code and I do a reformat, Idea doesn't change it. So please revert it.